### DISCUSSION TITLE
Counter in dataviewer

### DISCUSSION THREAD

**Discussion by KrisvanderMast** [microsoft/vscode-jupyter#11081] [New Ideas]
When using the dataviewer in Jupyter notebook I can easily filter. That's great but I can't see how many records remain of the total when filtering. For example if I have 30.000 records displayed and filter on a column first name for Kris I'd love to see how many records have Kris in their first name column. 

This discussion started first out on Twitter: https://twitter.com/KrisvanderMast/status/1554898271172272133

**Comment by IanMatthewHuff** [comment]
@KrisvanderMast This is a solid suggestion, thanks for filing it. If you don't mind. I'm actually going to convert this discussion into an issue. Discussions we use a bit more for "how to use feature X" type of requests and things that don't really have a single fix that we can make. This suggestion would get tracked best as a feature enhancement issue.

**Reply by IanMatthewHuff** [reply]
Track this issue here if you interested in tracking when this gets looked at: https://github.com/microsoft/vscode-jupyter/issues/11082
